changelogs/unreleased/6211-support-not-in,.yml

```diff
@@ -0,0 +1,6 @@
+description: Add the syntactic sugar "value not in items" for "not (value in items)"
```

Comment:
I'm not sure the "syntactic sugar" part is relevant for the end user. I'd just call it the `not in` operator or something.